pkgs/applications/audio/plexamp/default.nix

```diff
@@ -2,24 +2,25 @@
 
 let
   pname = "plexamp";
-  version = "4.8.3";
+  version = "4.10.1";
 
   src = fetchurl {
     url = "https://plexamp.plex.tv/plexamp.plex.tv/desktop/Plexamp-${version}.AppImage";
-    name="${pname}-${version}.AppImage";
-    hash = "sha512-CrSXmRVatVSkMyB1QaNSL/tK60rQvT9JraRtYYLl0Fau3M1LJXK9yqvt77AjwIwIvi2Dm5SROG+c4rA1XtI4Yg==";
+    name = "${pname}-${version}.AppImage";
+    hash = "sha512-Y43W6aPnZEMnkBQwSHbT8PWjuhkfNxce79Y9cixEPnq4VybROHGb9s6ghV7kP81TSWkb9QruFhmXfuJRSkXxTw==";
   };
 
   appimageContents = appimageTools.extractType2 {
     inherit pname version src;
   };
-in appimageTools.wrapType2 {
+in
+appimageTools.wrapType2 {
   inherit pname version src;
 
   extraInstallCommands = ''
     install -m 444 -D ${appimageContents}/plexamp.desktop $out/share/applications/plexamp.desktop
-    install -m 444 -D ${appimageContents}/plexamp.png \
-      $out/share/icons/hicolor/512x512/apps/plexamp.png
+    install -m 444 -D ${appimageContents}/plexamp.svg \
+      $out/share/icons/hicolor/512x512/apps/plexamp.svg
```

Comment:
```suggestion
      $out/share/icons/hicolor/scalable/apps/plexamp.svg
```

Reply:
Was there any testing on this? why would you make this change, if this is wrong it will cause the package to fail to build.

Reply:
Please read https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html